azext_iot/tests/iothub/test_iot_messaging_int.py

```diff
@@ -18,7 +18,7 @@
 )
 
 settings = DynamoSettings(ENV_SET_TEST_IOTHUB_BASIC)
-LIVE_HUB = settings.env.azext_iot_testhub
+LIVE_HUB = settings.env.azext_iot_testhub if settings.env.azext_iot_testhub else "test-hub-" + str(uuid4())
```

Comment:
Instead of `str(uuid4())` you can use `...tests.generators.generate_generic_id()` which we are trying to maintain usage on when this respective need arises.

Reply:
Makes sense, will make the change